src/Integration.UnitTests/LocalServices/ErrorListInfoBarControllerTests.cs

```diff
@@ -182,7 +184,7 @@ public void ErrorListInfoBarController_Refresh_Connected_ActiveSolutionBoundAndF
         }
 
         [TestMethod]
-        public void ErrorListInfoBarController_Refresh_ActiveSolutionBound_NotFullyLoaded_HasUnboundProjects()
+        public async Task ErrorListInfoBarController_Refresh_ActiveSolutionBound_NotFullyLoaded_HasUnboundProjects()
```

Comment:
The product code now contains an `async` class so some of the tests need to be changed to be `async` too.